tests/loggers/test_base.py

```diff
@@ -176,6 +176,32 @@ def training_step(self, batch, batch_idx):
     assert logger2.finalized_status == "success"
 
 
+def test_trainer_loggers_property(tmpdir):
+    logger1 = CustomLogger()
+    logger2 = CustomLogger()
+
+    # trainer.loggers should be a copy of the input list
+    trainer = Trainer(max_steps=2, log_every_n_steps=1, logger=[logger1, logger2], default_root_dir=tmpdir)
```

Comment:
why do you set `max_steps`, `log_every_n_steps` and `default_root_dir` if the `Trainer` will not run?

Reply:
Good point. I had just refactored the previous test and wasn't sure if it was necessary. It's now been removed.